src/python/bot/fuzzers/afl/engine.py

```diff
@@ -82,13 +135,21 @@ def reproduce(self, target_path, input_path, arguments, max_time):
     Returns:
       A ReproduceResult.
     """
-    config = launcher.AFLConfig.from_target_path(target_path)
+    environment.set_value('AFL_ENGINE_TIMEOUT', max_time)
```

Comment:
this doesn't do anything in the reproduce case right?

Reply:
+1

Reply:
Right, removed.